modules/oc-adm-by-example-content.adoc

```diff
@@ -262,6 +262,18 @@ Collect debugging data for a given resource
 
 
 
+== oc adm migrate icsp
+Update imagecontentsourcepolicy file(s) to imagedigestmirrorset file(s).
```

Comment:
To align with [the docs](https://docs.openshift.com/container-platform/4.13/post_installation_configuration/preparing-for-users.html#images-configuration-registry-mirror-convert_post-install-preparing-for-users) on this:
s/imagecontentsourcepolicy/`ImageContentSourcePolicy` 
s/imagedigestmirrorset/`Image Digest Mirror Set`